omod/src/main/webapp/pages/allergies.gsp

```diff
@@ -84,7 +84,7 @@ ${ ui.includeFragment("uicommons", "infoAndErrorMessage")}
 	                <td>
 	                	<i class="icon-pencil edit-action" title="${ ui.message("coreapps.edit") }"
 	                       onclick="location.href='${ ui.pageLink("allergyui", "allergy", [allergyId:allergy.id, patientId: patient.id, returnUrl: returnUrl]) }'"></i>
-	                    <i class="icon-remove delete-action" title="${ ui.message("coreapps.delete") }" onclick="removeAllergy('${ allergy.allergen }', ${ allergy.id})"></i>
+	                    <i class="icon-remove delete-action" title="${ ui.message("coreapps.delete") }" onclick="removeAllergy('${ allergy.allergen.escapeJavascript }', ${ allergy.id})"></i>
```

Comment:
Yes, I did. Keep in mind that this change depends on:
https://github.com/openmrs/openmrs-module-allergyapi/pull/17